schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+{
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Instrument Specification Definition",
+   "description": "The Instrument Specification Definition for Astro CSM",
+   "type": "object",
+
+   "properties": {
+       "BORESIGHT": {
+	   "description": "Boresight (x,y,z) vector",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "description": "Component of the boresight vector",
+		 "minItems": 3,
+                 "maxItems": 3
+            }
+       },
+       "DETECTOR_CENTER": {
+           "description": "Center of the detector in pixel coordinates",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 2,
+                 "maxItems": 2
+            }
+       },
+       "CENTER_EPHEMERIS_TIME": {
+           "description": "The center ephemeris time of the image",
+           "type": "number"
+       },
+       "STARTING_EPHEMERIS_TIME": {
+           "description": "The starting ephemeris time of the image",
+           "type": "number"
+       },
+       "FOCAL_LENGTH_MODEL": {
+            "description":  "A key to get information about time/temp dependent focal length models",
+            "type": "object",
+            "properties": {
+                "key": {"type": "string"},
+                "focal_length": {"type": "number"},
+                "focal_epsilon":{"type": "number"}
+            },
+   	"required": ["focal_length"]
```

Comment:
Should "key" be required also if we're going to use it for parsing?  We could default it to constant if we don't want to require it.